src/webgpu/shader/execution/robust_access_vertex.spec.ts

```diff
@@ -207,181 +239,203 @@ class DrawCall {
       this.baseVertex,
```

Comment:
`generateIndexedIndirectBuffer` uses `firstVertex` instead of `firstIndex`

Reply:
Done.